CHANGELOG.md

```diff
@@ -4,6 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] 2024-11-25
+### Added
+- Add decryption option to allow disabling the integrity tag requirement.
+- Add option to decrypt with signature keys for legacy messages.
+- Fix naming `profile.Custom.AllowWeakRSA` to `profile.Custom.InsecureAllowWeakRSA` (breaking)
+
+### Changed
+- Update go-crypto to `1.1.3`.
```

Comment:
Actually it should probably go under Changed, and then it's also more obvious that it's a breaking change.
Also I wouldn't go so far as to call it a fix, the old name wasn't wrong per se, the new one is just more explicit :blush: 

```suggestion

### Changed
- Rename `profile.Custom.AllowWeakRSA` to `profile.Custom.InsecureAllowWeakRSA`
- Update go-crypto to `1.1.3`.
```